README.md

```diff
@@ -63,6 +63,18 @@ With these pieces of information on hand, you’re ready to have your end users
 https://dashboard.checkr.com/oauth/authorize/:client_id?scope=read_write
```

Comment:
Yep, the idea is to get everyone to switch to using `partners.checkr.com` directly. dashboard does a redirect to support partner that implemented oauth before the new flow